src/attest.ts

```diff
@@ -6,29 +6,34 @@ const OCI_TIMEOUT = 30000
 const OCI_RETRY = 3
 
 export type SigstoreInstance = 'public-good' | 'github'
+export type CreateAttestationOptions = {
+  sigstoreInstance: SigstoreInstance
+  pushToRegistry: boolean
+  githubToken: string
+}
 export type AttestResult = Attestation & {
   attestationDigest?: string
+  attestationSubjects: Subject[]
```

Comment:
We now record which subjects are included in each attestation.

Reply:
Now that I've read the whole PR, this is only ever used inside the log functions, right? If we rollback the `createMulti/Single` functions as I suggested then we don't need to change this result type.

Reply:
Reverted (but see other comment).